### DISCUSSION TITLE
Release 5.3.2 - updated dependencies

### DISCUSSION THREAD

**Discussion by miguel-a-calles-mba** [serverless-heaven/serverless-webpack#1545] [Team Posts]
Hi, I've been reviewing and merging PRs that update the dependencies.

I drafted the release.

https://github.com/serverless-heaven/serverless-webpack/releases/tag/untagged-54bd590e1d9f15338a68

Are there any other PRs we should consider for this release?

**Comment by jamesmbourne** [comment]
@miguel-a-calles-mba Thanks for putting that together - I think a patch with some dependency updates is a good starting point 👍.

Looking ahead to the next release, there is a handful of PRs that seemed to be frequently requested - I'd personally like to try and get #560 and #517 in for the next one.

**Comment by HyperBrain** [comment]
@miguel-a-calles-mba I added the NPM secret with publish rights to the repo named NPM_TOKEN.
Please check if it works properly.

**Comment by miguel-a-calles-mba** [comment]
Thank you @HyperBrain. I will test after merging https://github.com/serverless-heaven/serverless-webpack/pull/581.

May someone help double-check the release notes?

**Comment by jamesmbourne** [comment]
@miguel-a-calles-mba I've approved #581 but left it unmerged for now.

I've taken a look at https://github.com/serverless-heaven/serverless-webpack/compare/v5.3.1...master and spotted a few PRs that aren't in the draft release notes:

- https://github.com/serverless-heaven/serverless-webpack/pull/518
- https://github.com/serverless-heaven/serverless-webpack/pull/524
- https://github.com/serverless-heaven/serverless-webpack/pull/520
- https://github.com/serverless-heaven/serverless-webpack/pull/526
- https://github.com/serverless-heaven/serverless-webpack/pull/527

Is it worth including these in the notes?

**Comment by miguel-a-calles-mba** [comment]
Thanks @jamesmbourne! I added them to the release notes.

I gotta learn how to do that diff you sent. :)

**Comment by miguel-a-calles-mba** [comment]
I created the release PR. https://github.com/serverless-heaven/serverless-webpack/pull/582

Should we do any tests outside of those in the CI?

**Comment by jamesmbourne** [comment]
@miguel-a-calles-mba there doesn't seem to be an easy way to access it through the UI - appending `/compare` to the repo URL should let you do it though e.g. https://github.com/serverless-heaven/serverless-webpack/compare/

I'll pull down the branch and give it a try against a real AWS env - there are a lot of dependency changes so there is potential for issues to be introduced via that channel

**Comment by miguel-a-calles-mba** [comment]
Thanks!

**Comment by miguel-a-calles-mba** [comment]
I installed the master branch into a serverless stack. It deployed with no issues. I ran a regression test against it, and it passed. The build seems stable to me.

**Comment by miguel-a-calles-mba** [comment]
Are there any objections to making the release?

**Comment by miguel-a-calles-mba** [comment]
Here is the release PR: https://github.com/serverless-heaven/serverless-webpack/pull/582

**Comment by miguel-a-calles-mba** [comment]
I got two approvals on the release PR. I'm merging and trying the npm publish with GitHub Actions.

**Comment by miguel-a-calles-mba** [comment]
The GitHub Action worked, and release 5.3.2 is available in npm.

**Comment by miguel-a-calles-mba** [comment]
Thanks @HyperBrain for adding the NPM_TOKEN secret. The npm publish can be automated going forward.
